client-state/store.ts

```diff
@@ -94,6 +94,18 @@ export const RootStore = types
     activeTab: types.optional(types.number, 0),
     lang: types.enumeration(Object.values(Language)),
   })
+  .views((self) => ({
+    getTabNameForAnalytics(index: number) {
+      if (index == 0) {
+        return 'questions'
+      } else if (index == 1) {
+        return 'results'
+      } else if (index == 2) {
+        return 'faq'
+      }
+      return '' // TODO: throw? Or unknown?
```

Comment:
Leaning toward `''` being okay, if we can't tell which tab we're on we should still send a request to AA

Reply:
going to go with unknown, better debugging breadcrumb